### PR TITLE
Created README.md for Bug Busters Website repo

### LINKED ISSUE #1: Add a readme for the project
Add guidelines on how to run the project + how to contribute + info about the project admins.

### PR DESCRIPTION
**Pull request by muditchoudhary**
Closes #1

# What I have done
- Created a README.md for Bug Busters Website repo

# What I have added
- About Bug Busters community section
- How to contribute section
- How to run the project section
-  About Project Admin

# Additional Information
- There is one link **here are tutorials using GUI tools.** is not pointing to a tutorial but pointing to Embellish self repo. Is this a mistake? Can I add this https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop Getting started with GitHub Desktop docs link?

### CODE REVIEW

**Comment by siddhi-244** [comment]
> Closes #1
> 
> # What I have done
> * Created a README.md for Bug Busters Website repo
> 
> # What I have added
> * About Bug Busters community section
> * How to contribute section
> * How to run the project section
> * About Project Admin
> 
> # Additional Information
> * There is one link **here are tutorials using GUI tools.** is not pointing to a tutorial but pointing to Embellish self repo. Is this a mistake? Can I add this https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop Getting started with GitHub Desktop docs link?

That would be a mistake. You can change that

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
One more thing. Just add the technology stack in one line!

File: README.md [hunk 1]
```diff
@@ -1 +1,122 @@
...
+
+<hr>
+
+
+<h2 align=center> Project Maintainer & Admin </h2>
+
+<p align="center"> <a href="https://github.com/siddhi-244">
+<img src="https://user-images.githubusercontent.com/64035221/135665400-b5e1edbe-2164-414b-b4cc-7574d0cacaeb.png" width=100px height=100px />
+</a><p align="center"> Siddhi Bhanushali </p>
+
```
Comment (Aditi-Singh16) [inline README.md]: Add @Aditi-Singh16  as project maintainer too!

**Comment by muditchoudhary** [comment]
Hey!! admins made the requested changes.

**Review by siddhi-244 — APPROVED** [review]
LGTM 🥳

**Comment by Aditi-Singh16** [comment]
@muditchoudhary nice work!
merging :D

**Review by Aditi-Singh16 — APPROVED** [review]

**Comment by muditchoudhary** [comment]
> @muditchoudhary nice work! merging :D

Yayy!!! :) Thank you :blush:

**Pull request merged**